### ISSUE TITLE
[BUG] Mobile: Not Focusing on the First Field 

### ISSUE THREAD

**Issue by FelipeRearden** [danielo515/obsidian-modal-form#126]
Hello @danielo515 !!!!!

🆘 This BR happens when using Modal Form on Obsidian Mobile on iPadOS with an external mouse and keyboard.

Different from Desktop, when opening a Form (any of the two methods: in-line or standard) the first field of the form is NOT focused the same way that is focused on Desktop.

⭘ the no matter the type of the field (text or select);
⭘ the user need to hit TAB to focus the first field.

🦶 Steps on iPad

1️⃣ Open a Modal form with the first field as text or text area
2️⃣ You will see that there is no caret cursor and the field is not focused 
3️⃣ Hit TAB to get focus in the first field 

To see this BR in action, you have to be on Obsidian Mobile on iPad. if you emulate mobile on Desktop **this issue DOES NOT occur**

<hr>

Thanks for reading this !!!!! I hope you like my suggestion 🙏 🙏 🙏

Let me know if you need more information !!!

Have a great day 🌞

**Comment by hdsateyate** [comment]
YES! I'm so pleased to see this, I was going to report it myself... Same behaviour on android.

Would save me a lot of effort over the course of the 10-15 inputs I make each day if the first field was focused automatically!

Many thanks.

**Comment by AAraKKe** [comment]
I am using this in MacOS and I am having the same issue, not only on mobile.
